static/js/docdock.js

```diff
@@ -58,7 +58,7 @@ jQuery(document).ready(function () {
     });
 
     // Add link button for every
-    var text, clip = new Clipboard('.anchor');
+    var text, clip = new ClipboardJS('.anchor');
```

Comment:
https://github.com/zenorocha/clipboard.js/releases/tag/v2.0.0
This is the only breaking change